examples/heat_equation/source/heat_ida.cc

```diff
@@ -619,13 +624,17 @@ int Heat<dim>::solve_jacobian_system (const double t,
   computing_timer.enter_section ("   Solve system");
   set_constrained_dofs_to_zero(dst);
 
-
-  SolverControl solver_control (dof_handler->n_dofs(), 1e-8);
-
-  SolverCG<VEC> solver(solver_control,
-                       SolverCG<VEC>::AdditionalData(dof_handler->n_dofs(),true));
-  solver.solve (jacobian_matrix, dst, src,
-                TrilinosWrappers::PreconditionIdentity());
+  // Ainv.op   = linear_operator<VEC>( jacobian_matrix );
+  // Ainv.prec = linear_operator<VEC>( jacobian_matrix,
+  //                                   TrilinosWrappers::PreconditionIdentity() );
+  // Ainv.vmult(src, dst);
+  dst = Ainv * src;
```

Comment:
@luca-heltai 
```
error: invalid operands to binary expression ('const ParsedSolver<VEC>' and 'const VEC' (aka 'const dealii::TrilinosWrappers::MPI::Vector'))
  dst = Ainv * src;
```
What is the correct usage of this tool? I read the header file and it seems to be right.